.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "SpongeAPI"]
 	path = SpongeAPI
-	url = https://github.com/SpongePowered/SpongeAPI.git
+	url = https://github.com/kashike/SpongeAPI.git
```

Comment:
It's been a common pattern for pr's to point to the contributors own spongeapi branch, that way travis can build and check them correctly, however a "Do not merge this commit" in the message would have been nice.